### DISCUSSION TITLE
Please NO

### DISCUSSION THREAD

**Discussion by demarijm** [community/community#13669] [Other Feature Feedback, Questions, & Ideas] [answered]
Please don't introduce distraction into a platform I have to use for work.

**Comment by jfullstackdev** [comment, marked as answer]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by thoroc** [reply]
@jdevstatic can you default the landing to the Following tab instead of the For you?
